DXRMapFixups/DeusEx/Classes/DXRFixupM15.uc

```diff
@@ -531,6 +536,24 @@ function PreFirstEntryMapFixes_Page()
             break;
         }
     }
+
+    if (VanillaMaps) {
+        foreach AllActors(class'AmbientSound', as) {
+            if (as.AmbientSound == Sound'Ambient.Ambient.GeigerLoop') {
+                ast = as.Spawn(class'DXRAmbientSoundTrigger',, 'Gray_rad');
+                ast.AmbientSound = as.AmbientSound;
+                ast.SoundRadius = as.SoundRadius;
+                ast.SoundVolume = as.SoundVolume;
+                ast.SoundPitch = as.SoundPitch;
```

Comment:
what about the triggered values? also this would be a bit cleaner if you called copy to triggered, and then called trigger on it

Reply:
The triggered values don't matter here, because Trigger never gets called on this.

Reply:
oh yea you can't reenable the radiation lol

but it would remove the need to copy the values like this

Reply:
How would the need to copy the values like this get removed?

Reply:
Ah, I see what you meant now. I agree that this is cleaner.

```
                ast.CopyValsToTriggered(as);
                ast.Trigger(None, None);
                ast.CopyValsToUntriggered(ast);
                ast.UntriggeredSoundVolume /= 3;
```

Reply:
Actually, calling `Trigger` when creating it doesn't work because it results in `bAlreadyTriggered` being set to true.